collections/_news/2022-01-29-autumn-2021.md

```diff
@@ -0,0 +1,32 @@
+
+![](/assets/images/2021-autumn/mingmoon.jpg)
+_CSS enjoys a Chinese buffet in Birmingham City Centre_
+
+Throughout the rest of the Freshers’ Week, we challenged you with a scavenger hunt around campus. Questions ranged from finding the Alan Turing meeting room, to discovering that the University has tunnels underneath campus. This was the first chance for you to pick up one of our Old Joe themed TeX dino stickers, and we hope you learnt something about our campus too! Don’t worry if you missed out on our Old Joe stickers though, there’ll be plenty more chances to get your hands on one. 
+
+We continued strongly through October, and into November and December with lots more events. This semester saw a variety of social events, including our first pub night at the Bristol Pear, and a return to hosting the Halloween Quiz and Board Games Night with the School. The first semester also saw a series of workshops starting from Python basics, leading up to a Discord Bot hackathon. 2021 concluded for us with the Christmas Meal, now at the Country Girl pub after a two year hiatus. Throughout these few months, we also brought back pizza - with a total of over 600 slices so far.
+
+![](/assets/images/2021-autumn/halloween.jpg)
+_The Computer Science Atrium stuffed with teams trying to complete the Halloween Quiz_
+
+This year, we haven’t just stayed on the tech side of things; we’ve ramped up our efforts on the sport front: starting with not only bouldering sessions but also supporting two football teams in the Campus League. After a shaky start, both teams eventually beat the odds and won their last games of the semester. The 0th and 1st teams placed 8th and 7th in their leagues respectively. Well done and we’re looking forward to more successes in the series of games.
+
+![](/assets/images/2021-autumn/bouldering.jpg)
+_We ran plenty of Bouldering trips to Depot in Birmingham City Centre too_
+
+In 2019-20, CSS set its previous record for memberships at just under 300. This year, we are proud to announce that CSS has grown immensely and has surpassed our previous record after just 8 weeks. We end the Autumn semester with 323 members! This is a promising outlook and we can’t wait to see what records we have yet to beat in the rest of the year. 
+
+Going forward into the rest of the year, we have more great events lined up. From more events like socials and workshops, to our highlights of the year: a trip to Bletchley Park and the annual CSS Ball. I want to end with a massive thank you to all of our members for making the Society a huge success, and we look forward to seeing what the next semester has in store for CSS.
+
```

Comment:
Stylistic choice but adding a line break here

`---`

see one of the earlier posts.